pkg/kubelet/kubelet.go

```diff
@@ -120,7 +120,7 @@ const (
 
 	// Generic PLEG relies on relisting for discovering container events.
 	// The period directly affects the response time of kubelet.
-	plegRelistPeriod = time.Second * 3
+	plegRelistPeriod = time.Second * 1
```

Comment:
@yujuhong Sorry, "add a line note" is not my word, it's the github text I accidently copied when moving my comments from commits to here :)

But more docs are welcome. Did you do any experiments so you changed the default period?